src/main/java/com/powsybl/openloadflow/util/Reports.java

```diff
@@ -348,14 +348,9 @@ public static ReportNode createRootLfNetworkReportNode(int networkNumCc, int net
                 .build();
     }
 
-    public static ReportNode createLfNetworkReportNode(ReportNode reportNode, ReportNode lfNetworkReportNode, int networkNumCc, int networkNumSc) {
-        ReportNode newReportNode = reportNode.newReportNode()
-                .withMessageTemplate("lfNetwork", "Network CC${networkNumCc} SC${networkNumSc}")
-                .withUntypedValue(NETWORK_NUM_CC, networkNumCc)
-                .withUntypedValue(NETWORK_NUM_SC, networkNumSc)
-                .add();
-        newReportNode.include(lfNetworkReportNode);
-        return newReportNode;
+    public static ReportNode includeLfNetworkReportNode(ReportNode reportNode, ReportNode lfNetworkReportNode) {
+        reportNode.include(lfNetworkReportNode);
+        return lfNetworkReportNode;
     }
```

Comment:
@olperr1 why is this change part of an integration branch ? I didn't find the API change in core that requries to change this behaviour.

Reply:
It's an adaptation for https://github.com/powsybl/powsybl-core/pull/3007.
When a `ReportNode` was included, its root was omitted and only its children were in fact included. The powsybl-core PR fixed that, so I reverted the workaround that was made in OLF.

Reply:
Yes indeed at the time I made this code in #919 I went a bit fast and didn't fully realize it was actually a powsybl-core bug I was workaround-ing here ...

Only when I saw https://github.com/powsybl/powsybl-core/pull/3007 I understood this wasn't right and noted here: https://github.com/powsybl/powsybl-open-loadflow/pull/919#discussion_r1602876818

thanks @olperr1 for the fix